gitdiff/parser.go

```diff
@@ -15,7 +16,7 @@ const commitPrefix = "commit"
 // Parse parses a patch with changes to one or more files. Any content before
 // the first file is returned as the second value. If an error occurs while
 // parsing, it returns all files parsed before the error.
-func Parse(r io.Reader) (<-chan *File, error) {
+func Parse(cmd *exec.Cmd, r io.ReadCloser) (<-chan *File, error) {
```

Comment:
Worth calling out that this changes the API so that go-gitdiff can clean up zombie processes after it finishes parsing the stdout stream.